File: stubs/python/stub.py
# -*- coding: utf-8 -*-
from hornero import *
# proyecto para python3
# necesita la libreria python3-httplib2. Instalación en linux >sudo apt-get install python3-httplib2
# modificar el token que le corresponde al equipo para el torneo
# y el numero de Problema a Trabajar

token ='d3d2b9557a28de20132a112c7034379f'
problema = '1'
servidor= Hornero()
parametros= servidor.solicitud(token,problema)
# Se piden los parametros al servidor hornero NO TOCAR!!!!

#-------------------------------------------------------------------
#Resolver el ejercicio tomando los parámetros del array parametros en formato str
#y asingar la respuesta a la variable respuesta tambien en formato str

x= int(parametros[0])
y= int(parametros[1])

respuesta=str(x+y)

#-------------------------------------------------------------------
#Se envía la respuesta al servidor hornero NO TOCAR!!!!
servidor.envio(respuesta)

File: stubs/php/stub.php
<html>
    <meta charset="UTF8">

    <body>
<?php
require_once './hornero.php';
/*
modificar el token que le corresponde al equipo para el torneo
y el numero de Problema a Trabajar
*/
$token='85bebdbee1c4159013d7af54f22f3f1e';
$problema=2;
$hornero=new Hornero($problema,$token);
$hornero->host='localhost/yii/hornero';
$parametros=$hornero->solicitud();
/*******************************************************************
Resolver el ejercicio tomando los parámetros del array parametros
y asingar la respuesta a la variable respuesta
*/

$x=$parametros[0];
$y=$parametros[1];
$respuesta= $x+$y;

/*******************************************************************
Se envía la respuesta al servidor hornero NO TOCAR!!!!*/
$respuesta=$hornero->respuesta($respuesta);

print_r($respuesta);
?>
</body>
</html>